source/_docs/mqtt/broker.markdown

```diff
@@ -92,6 +92,31 @@ mqtt:
   certificate: /home/paulus/downloads/mosquitto.org.crt
 ```
 
+### HiveMQ Cloud
+
+HiveMQ Cloud is a fully managed MQTT broker that provides you a private broker. A free plan for up to 100 devices is available.
```

Comment:
Let's make sure we link to the page that makes that clear and also shows the limitation the account brings.

Additionally, we don't recommend using the cloud. We should make sure it is clear that we strongly recommend using a local broker.